rotkehlchen/externalapis/routescan.py

```diff
@@ -0,0 +1,54 @@
+import logging
+from typing import TYPE_CHECKING, Final
+
+from rotkehlchen.errors.misc import ChainNotSupported
+from rotkehlchen.externalapis.etherscan_like import EtherscanLikeApi
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import SUPPORTED_CHAIN_IDS, ApiKey, ChainID, ExternalService
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+    from rotkehlchen.user_messages import MessagesAggregator
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+ROUTESCAN_PAGINATION_LIMIT: Final = 10000
+ROUTESCAN_BASE_URL: Final = 'https://api.routescan.io/v2/network/mainnet/evm/{chain_id}/etherscan/api'
```

Comment:
question: I guess `mainnet` means that is for example BASE mainnet but how does it work? What are the possible values? Can you link to the explanation where this is documented?